optaplanner-core/src/test/java/org/optaplanner/core/impl/heuristic/selector/move/generic/chained/TailChainSwapMoveTest.java

```diff
@@ -278,4 +280,39 @@ public void toStringTest() {
                 inverseVariableSupply, anchorVariableSupply, a2, a0).toString());
     }
 
+    @Test
+    public void retrievedPlanningEntitiesContainNeverNull() {
```

Comment:
It's nice if the test method name starts with (or equals) the method being tested.
Would you mind renaming it to getPlanningEntitiesWithRightEntityNull() ?